tests/utils/gossamer_utils.go

```diff
@@ -285,10 +285,11 @@ func InitNodes(num int, config string) ([]*Node, error) {
 
 // StartNodes starts given array of nodes
 func StartNodes(t *testing.T, nodes []*Node) error {
-	for _, n := range nodes {
+	for i, n := range nodes {
 		err := StartGossamer(t, n, false)
 		if err != nil {
-			return nil
+			return fmt.Errorf("node %d of %d: %w",
+				i+1, len(nodes), err)
```

Comment:
I guess that's fine to return an error here if we fail to start gossamer in a test right?

Reply:
yeah that's fine